README.md

```diff
@@ -1,28 +1,59 @@
 # NaturalDateFormat
 
-A library to format dates as described in Material Design guidelines (https://www.google.com/design/spec/patterns/data-formats.html#data-formats-date-time).
+A HMOS library to format dates. RelativeDateFormat formats date as relative (4 months ago, today, next year), AbsoluteDateFormat formats date as absolute (Monday, 4:32 PM, June 10).
```

Comment:
"RelativeDateFormat formats date as relative (4 months ago, today, next year), AbsoluteDateFormat formats date as absolute (Monday, 4:32 PM, June 10)." can be added in the Features Section. Also explain about org.joda time lib that is used in the library.

Reply:
done